integrationTests/nodesCoordinatorFactory.go

```diff
@@ -112,6 +114,8 @@ func (ihncrf *IndexHashedNodesCoordinatorWithRaterFactory) CreateNodesCoordinato
 		MaxNodesEnableConfig: nil,
 		EnableEpochsHandler: &testscommon.EnableEpochsHandlerStub{
 			IsBalanceWaitingListsFlagEnabledField: true,
+			IsStakingV4Step2FlagEnabledField:      true,
+			IsStakingV4Step3FlagEnabledField:      true,
```

Comment:
[question]

Were tests not passing because of this?

Reply:
rechecked, not needed, removed